.github/workflows/ci.yml

```diff
@@ -66,8 +66,12 @@ jobs:
         pip list
         sudo apt-get update
         sudo apt-get -qq install pandoc
+    - name: Check docstrings
+      run: |
+        pydocstyle --select D102 src/pyhf/pdf.py
```

Comment:
D102 is "missing docstring" I propose we enable other error codes only once all src files pass D102